src/base/client.zig

```diff
@@ -416,7 +425,7 @@ test "attempt echo on echo.websocket.org" {
     var masked: [4]u8 = undefined;
     client.maskPayload("test", &masked);
 
-    try client.writeMessagePayload(&masked);
+    try client.writeUnmaskedPayload(&masked);
```

Comment:
This test should probably use `writeMessagePayload` instead of manually masking.